partials/agechecker-form.php

```diff
@@ -1,22 +1,39 @@
-<form id="age-checker-form">
+<div class="overlay-position" id="age-overlay">
 
-  <p>Please enter your date of birth:<p>
-  
-  <div>
-    <input id="birthday-month" name="month" value="" type="number" min="1" max="12" placeholder="MM">
-  </div>
+  <div class="container-blur"></div>
 
-  <div>
-     <input id="birthday-day" name="day" value="" type="number" min="1" max="31" placeholder="DD">
-  </div>
+  <div class="container-overlay grid-row justify-center">
```

Comment:
i think we're better to use an `id` here. maybe `id="age-checker-wrapper"` would be more semantic